issuesync.rb

```diff
@@ -80,11 +80,14 @@ def format(io, issue, comments)
 
     def format_body(io, issue)
       title = "##{issue.number}: #{issue.title.strip}"
+      title << issue.labels
```

Comment:
To avoid the first line of the file becoming too long, should we consider adding labels on a separate line?